app/views/step_by_step_pages/show.html.erb

```diff
@@ -18,6 +18,19 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% unless @step_by_step_page.can_be_published? %>
+      <%= render "govuk_publishing_components/components/inset_text" do %>
+        <h2 class="govuk-heading-m">To publish this step by step you need to</h2>
+        <% if !@step_by_step_page.steps.any? %>
+          <a href="<%= new_step_by_step_page_step_path(@step_by_step_page) %>">Add at least one step</a>
```

Comment:
I suggest we use the Rails helpers (`link_to`) consistently to generate links and add `govuk-link` class to them.